code/modules/uplink/uplink_items/uplink_badass.dm

```diff
@@ -84,11 +77,3 @@
 	limited_stock = 1
 	cant_discount = TRUE
 	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
-
-/datum/uplink_item/badass/shades
```

Comment:
This one is used a bit, can we keep it?

Reply:
They are not even that much badass, specially when we have gar glasses anyway.